### PR TITLE
Add Network ACL Administrators group and policies

### PR DESCRIPTION
**Pull request by RichardWeiss**
Added group and policies to allow IAM users to administer NACLs.

### CODE REVIEW

**Comment by gozer** [comment]
Needs a corresponding issue

File: vpc/vpc-meta.template [hunk 1]
```diff
@@ -226,6 +226,34 @@
...
       }
     },
+    "NACLAdminGroup": {
+      "Type": "AWS::IAM::Group",
+      "Properties": {
+        "NACLAdminPolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+        {
+            "Sid": "Stmt1445360254279",
```
Comment (gozer) [inline vpc/vpc-meta.template]: This is generated and doesn't need to be present in clean templates

File: vpc/vpc-meta.template [hunk 2]
```diff
@@ -226,6 +226,34 @@
...
+            "Action": [
+                "ec2:CreateNetworkAcl",
```
Comment (gozer) [inline vpc/vpc-meta.template]: I suspect this permission isn't necessary, the ACL is already created.

File: vpc/vpc-meta.template [hunk 3]
```diff
@@ -226,6 +226,34 @@
...
+                "ec2:CreateNetworkAclEntry",
+                "ec2:DeleteNetworkAcl",
```
Comment (gozer) [inline vpc/vpc-meta.template]: I don't think we want the MOC to have the ability to delete an entire ACL

File: vpc/vpc-meta.template [hunk 4]
```diff
@@ -226,6 +226,34 @@
...
+                "ec2:DeleteNetworkAclEntry",
+                "ec2:DescribeNetworkAcls",
+                "ec2:ReplaceNetworkAclAssociation",
```
Comment (gozer) [inline vpc/vpc-meta.template]: If all that's needed is the ability to add entries to an existing ACL, I don't see how that privilege is needed.

File: vpc/vpc-meta.template [hunk 5]
```diff
@@ -226,6 +226,30 @@
         ]
       }
     },
+    "NACLAdminGroup": {
+      "Type": "AWS::IAM::Group",
+      "Properties": {
+        "NACLAdminPolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+        {
```
Comment (limed) [inline vpc/vpc-meta.template]: Some formatting issue here, needs to be indented properly

Comment (limed) [inline vpc/vpc-meta.template]: Recommend running it through jq

File: vpc/vpc-meta.template [hunk 6]
```diff
@@ -226,6 +226,30 @@
...
+                "ec2:DescribeVpcAttribute",
+                "ec2:DescribeVpcs"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+                }
+              ]
+            }
+          }
+        ]
```
Comment (tinnightcap) [inline vpc/vpc-meta.template]: this brace does not match, same with one curly brace:

parse error: Unmatched ']' at line 250, column 11

**Pull request merged**